humanize/locale/fa_IR/LC_MESSAGES/humanize.po

```diff
@@ -0,0 +1,264 @@
+msgctxt "3"
+msgid "rd"
+msgstr "سومین"
+
+#: humanize/number.py:24
+msgctxt "4"
+msgid "th"
+msgstr "چهارمین"
+
+#: humanize/number.py:25
+msgctxt "5"
+msgid "th"
+msgstr "پنجمین"
+
+#: humanize/number.py:26
+msgctxt "6"
+msgid "th"
+msgstr "ششمین"
+
+#: humanize/number.py:27
+msgctxt "7"
+msgid "th"
+msgstr "هفتمین"
+
+#: humanize/number.py:28
+msgctxt "8"
+msgid "th"
+msgstr "هشتمین"
+
+#: humanize/number.py:29
+msgctxt "9"
+msgid "th"
+msgstr "نهمین"
+
+#: humanize/number.py:55
+msgid "million"
+msgstr "میلیون"
+
+#: humanize/number.py:55
+msgid "billion"
+msgstr "میلیارد"
+
+#: humanize/number.py:55
+msgid "trillion"
+msgstr "ترلیون"
+
+#: humanize/number.py:55
+msgid "quadrillion"
+msgstr "کوادریلیون"
+
+#: humanize/number.py:56
+msgid "quintillion"
+msgstr "کوانتیلیون"
+
+#: humanize/number.py:56
+msgid "sextillion"
+msgstr "سکستیلیون"
+
+#: humanize/number.py:56
+msgid "septillion"
+msgstr "سپتیلیون"
+
+#: humanize/number.py:57
+msgid "octillion"
+msgstr "اوکتیلیون"
+
+#: humanize/number.py:57
+msgid "nonillion"
+msgstr "نونیلیون"
+
+#: humanize/number.py:57
+msgid "decillion"
+msgstr "دسیلیون"
+
+#: humanize/number.py:57
+msgid "googol"
+msgstr "گوگول"
+
+#: humanize/number.py:91
+msgid "one"
+msgstr "یک"
+
+#: humanize/number.py:91
+msgid "two"
+msgstr "دو"
+
+#: humanize/number.py:91
+msgid "three"
+msgstr "سه"
+
+#: humanize/number.py:91
+msgid "four"
+msgstr "چهار"
+
+#: humanize/number.py:91
+msgid "five"
+msgstr "پنج"
+
+#: humanize/number.py:91
+msgid "six"
+msgstr "شش"
+
+#: humanize/number.py:92
+msgid "seven"
+msgstr "هفت"
+
+#: humanize/number.py:92
+msgid "eight"
+msgstr "هشت"
+
+#: humanize/number.py:92
+msgid "nine"
+msgstr "نه"
+
+#: humanize/time.py:64 humanize/time.py:126
+msgid "a moment"
+msgstr "یک لحظه"
+
+#: humanize/time.py:66
+msgid "a second"
+msgstr "یک ثانیه"
+
+#: humanize/time.py:68
+#, python-format
+msgid "%d second"
+msgid_plural "%d seconds"
+msgstr[0] "%d ثانیه"
+msgstr[1] "%d ثانیه"
+
+#: humanize/time.py:70
+msgid "a minute"
+msgstr "یک دقیقه"
+
+#: humanize/time.py:73
+#, python-format
+msgid "%d minute"
+msgid_plural "%d minutes"
+msgstr[0] "%d دقیقه"
+msgstr[1] "%d دقیقه"
+
+#: humanize/time.py:75
+msgid "an hour"
+msgstr "یک ساعت"
+
+#: humanize/time.py:78
+#, python-format
+msgid "%d hour"
+msgid_plural "%d hours"
+msgstr[0] "%d ساعت"
+msgstr[1] "%d ساعت"
+
+#: humanize/time.py:81
+msgid "a day"
+msgstr "یک روز"
+
+#: humanize/time.py:83 humanize/time.py:86
+#, python-format
+msgid "%d day"
+msgid_plural "%d days"
+msgstr[0] "%d روز"
+msgstr[1] "%d روز"
+
+#: humanize/time.py:88
+msgid "a month"
+msgstr "یک ماه"
+
+#: humanize/time.py:90
+#, python-format
+msgid "%d month"
+msgid_plural "%d months"
+msgstr[0] "%d ماه"
+msgstr[1] "%d ماه"
+
+#: humanize/time.py:93
+msgid "a year"
+msgstr "یک سال"
+
+#: humanize/time.py:95 humanize/time.py:103
+#, python-format
+msgid "1 year, %d day"
+msgid_plural "1 year, %d days"
+msgstr[0] "۱ سال و %d روز"
+msgstr[1] "ein Jahr und %d Tage"
```

Comment:
Looks like some German still here

Reply:
I'll update this to match `msgstr[0]` so it can be merged. New PR welcome to update it, if needed.

```suggestion
msgstr[1]  "۱ سال و %d روز"
```